### PR TITLE
[17.0][FIX] define openupgradelib git url in test-requirements.txt

### PR DESCRIPTION
**Pull request by legalsylvain**
Ref : https://github.com/OCA/OpenUpgrade/pull/4294

### CODE REVIEW

File: openupgrade_scripts/README.rst [hunk 1]
```diff
@@ -16,14 +16,14 @@ Openupgrade Scripts
 .. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
-.. |badge3| image:: https://img.shields.io/badge/github-OCA%2FOpenUpgrade-lightgray.png?logo=github
-    :target: https://github.com/OCA/OpenUpgrade/tree/17.0/openupgrade_scripts
-    :alt: OCA/OpenUpgrade
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fopenupgrade-lightgray.png?logo=github
+    :target: https://github.com/OCA/openupgrade/tree/17.0/openupgrade_scripts
```
Comment (pedrobaeza) [inline openupgrade_scripts/README.rst]: Why this change in casing? It was correct before

Comment (legalsylvain) [reply openupgrade_scripts/README.rst]: I didn't do that. Pre-commit did it.
Something should be wrong in readme generation. 

I'll take a look.

Comment (pedrobaeza) [reply openupgrade_scripts/README.rst]: Maybe the folder where you download this is in lower case?

Comment (legalsylvain) [reply openupgrade_scripts/README.rst]: > Maybe the folder where you download this is in lower case?

renamed into Camelcase, and doesn't change anything.

Comment (pedrobaeza) [reply openupgrade_scripts/README.rst]: Here it is the problem: https://github.com/OCA/OpenUpgrade/blob/843d412c7f62e58227249051cb226d7e6788dccc/.copier-answers.yml#L20
